.studio/common

```diff
@@ -3,6 +3,28 @@
 # This file is the place we will put all the common functionality that
 # two or more components share/require (ex. install elasticsearch)
 
+document "clean_all_compiled_protos" <<DOC
+  Delete all the files with extensions matching '*pb*go' in the repo (except in vendor/)
+DOC
+function clean_all_compiled_protos() {
+  install_if_missing core/git git
+
+  find /src/{api,components,lib} -name '*pb*go' -exec rm '{}' \; 
+  git clean -fdx
```

Comment:
This is what I'm guessing @msorens hit.  If we leave it we should probably put a warning in the documentation.